showCrime/showCrime/settings.py

```diff
@@ -1,3 +1,12 @@
+"""
+django app for https://OakCrime.org
```

Comment:
This docstring isn't providing developers with any new information. Anyone looking at this file probably started with the README at the root of the repo. This information should be in the README.